addons/hr_recruitment_survey/views/hr_job_views.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <record id="test_survey_view" model="ir.actions.act_window">
+        <field name="name">Survey</field>
```

Comment:
Please follow the guidelines.

https://www.odoo.com/documentation/13.0/reference/guidelines.html#xml-ids-and-naming